app/helpers/product_drive_helper.rb

```diff
@@ -4,4 +4,8 @@ def is_virtual(product_drive:)
 
     product_drive.virtual? ? 'Yes' : 'No'
   end
+
+  def can_delete_product_drive?(user, product_drive)
+    user.has_role?(Role::ORG_ADMIN, product_drive.organization) && product_drive.donations.empty?
```

Comment:
I'm a bit confused here. Can we instead move this to a policy object? Maybe under `app/policies`? We can pass in the product drive and the user. Then we can reuse this method across both the view and the backend. Something like:

```ruby
module ProductDrivePolicy
  class << self
    def can_destroy?(product_drive, user)
      user.has_role?(Role::ORG_ADMIN, product_drive.organization) && product_drive.donations.empty?
    end
  end
end
```

and then you can call `ProductDrivePolicy.can_destroy?(product_drive, user)` in both places.

I know we don't have any of these policy objects yet, but I think I like this pattern for future similar use cases as well.